packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -145,20 +145,20 @@ export class Sampler extends ThreeDOMElement implements SamplerInterface {
     this[$setProperty]('rotation', rotation);
   }
 
-  setScale(scale: Vector2|null): void {
+  setScale(scale: Vector2Interface|null): void {
     if (scale == null) {
       // Reset scale.
       scale = new Vector2(1, 1);
     }
-    this[$setProperty]('repeat', scale);
+    this[$setProperty]('repeat', new Vector2(scale.x, scale.y));
```

Comment:
Do we have an example that tests this? Did it work before?

Reply:
Yes we have a transform Textures example. It did work before on android and Mac, still works. I need to test with iOS.

Reply:
Wow, I guess three.js must automatically turn a scalar into a vector?